vendor/engines/production_credits/app/models/production_credits/production.rb

```diff
@@ -12,7 +12,7 @@ def category_enum
     end
 
     def name
-      "#{production_name} - #{open_on.year}"
+      "#{production_name} - #{open_on.try(:year)}"
```

Comment:
Do we really want a `nil` showing up wherever name is shown?  Maybe something like:

```ruby
if open_on
  "#{production_name} - #{open_on.year}"
else
  "#{production_name} - unknown year"
end
```